### PR TITLE
Add the bounding box of individual digits to the items the OCR returns.

### PR DESCRIPTION
**Pull request by dxaen**

### CODE REVIEW

File: CardScan/Classes/OcrDD.swift [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Foundation
 
 
 public class OcrDD{
+    public var scanStats = ScanStats()
```
Comment (kingst) [inline CardScan/Classes/OcrDD.swift:12]: instead of using scanstats for this why not create a variable that's just for a list of boxes?

Comment (dxaen) [reply CardScan/Classes/OcrDD.swift:12]: This is consistent with how Ocr.swift does it, incase we want to add more stuff to this. But I can create a variable specific to this if you prefer.

Comment (kingst) [reply CardScan/Classes/OcrDD.swift:12]: Yeah, OCR using scanstats is a bit of a throwback to the old way of doing it -- we track the real scan stats in the main loop class. Let's go with a specific variable.

Comment (dxaen) [reply CardScan/Classes/OcrDD.swift:12]: Alright.

File: CardScan/Classes/DetectedAllOcrBoxes.swift [hunk 2]
```diff
@@ -15,4 +15,8 @@ public struct DetectedAllOcrBoxes {
         let frameArray = self.allBoxes.map{$0.toDict()}
         return frameArray
     }
+    
+    public func getBoundingBoxesOfDigits() -> [CGRect] {
+        return self.allBoxes.map{$0.rect}
```
Comment (jaimejiyepark) [inline CardScan/Classes/DetectedAllOcrBoxes.swift:20]: nit: need spaces. `self.allBoxes.map { $0.rect }`

File: CardScan/Classes/OcrDD.swift [hunk 3]
```diff
@@ -9,6 +9,7 @@ import Foundation
```
Comment (jaimejiyepark) [inline CardScan/Classes/OcrDD.swift:10]: nit: get rid of one new line

File: CardScan/Classes/OcrDD.swift [hunk 4]
```diff
@@ -9,6 +9,7 @@ import Foundation
 
 
 public class OcrDD{
+    public var lastDetectedBoxes: [CGRect] = []
     public init(){}
```
Comment (jaimejiyepark) [inline CardScan/Classes/OcrDD.swift:13]: do we need this init if it's not used?

**Comment by jaimejiyepark** [comment]
sorry about all the nits! besides them, looks good! #shipit

File: CardScan/Classes/DetectedAllOcrBoxes.swift (hunk 2 above)
Comment (dxaen) [reply CardScan/Classes/DetectedAllOcrBoxes.swift:20]: yes ma'am.

File: CardScan/Classes/OcrDD.swift (hunk 3 above)
Comment (dxaen) [reply CardScan/Classes/OcrDD.swift:10]: 👍

File: CardScan/Classes/OcrDD.swift (hunk 4 above)
Comment (dxaen) [reply CardScan/Classes/OcrDD.swift:13]: I need to add some code here as in Ocr.swift. Thanks for pointing this out.

**Pull request merged**